src/Symfony/Component/Console/Helper/Helper.php

```diff
@@ -39,20 +39,6 @@ public function getHelperSet()
         return $this->helperSet;
     }
 
-    /**
-     * Returns the length of a string, using mb_strwidth if it is available.
-     *
-     * @deprecated since 5.3
-     *
-     * @return int The length of the string
-     */
-    public static function strlen(?string $string)
```

Comment:
this deserve an entry in the CHANGELOG (same for other changes)

Reply:
I added all changes as CHANGELOG entries: https://github.com/symfony/symfony/pull/41312/commits/8dd734ada31a7185fb1c63549a416d56bac262fa